packages/perseus/src/widgets/__tests__/__snapshots__/interactive-graph.test.ts.snap

```diff
@@ -352,26 +338,26 @@ exports[`interactive-graph widget question Shoud render predictably: after inter
                 </span>
                 <span
                   class="graphie-label"
-                  data-math-formula="5.0"
-                  style="position: absolute; padding: 7px; color: rgb(100, 149, 237); left: 269.14285714285717px; top: 126.85714285714285px; fill: none; stroke: #6495ED;"
```

Comment:
This probably changed from moving from unlimited points to a limited number of points